network/templates/base/stations.html

```diff
@@ -33,9 +35,9 @@
               </a>
             </td>
             <td>{{ station.name }}</td>
-            {% if station.location %}
+            {% if station.qthlocator %}
               <td title="{{ station.lat|floatformat:-3 }}, {{ station.lng|floatformat:-3 }}">
-                {{ station.location }}
+                {{ station.qthlocator }}
```

Comment:
So here you can just an extra column for `qthlocator` and rename `location` with `coordinates`. And adjust accordingly the `<thead>` section.